schematics/web/authentication/files/src/awsConfig.ts.template

```diff
@@ -6,6 +6,12 @@ export const AUTH_CONFIG = {
     region: process.env.REACT_APP_REGION,
     userPoolId: process.env.REACT_APP_USER_POOL_ID,
     userPoolWebClientId: process.env.REACT_APP_USER_POOL_WEB_CLIENT_ID,
+    oauth: {
```

Comment:
Will `oauth` be optional? If it will be, we'll need to put this code here based on some user input conditions

Reply:
Comment above fits to multiple places in a code below, so I won't duplicate it there

Reply:
Fixed by conditionally generating files associated with environment variables.
https://github.com/codeandpepper/janush/pull/230/commits/fefaed3409fcccf5406e1a7dae4cbec1ea2a60cf